src/Fable.Cli/Fable.Cli.fsproj

```diff
@@ -48,6 +48,7 @@
     <Compile Include="FileWatchers.fsi" />
     <Compile Include="FileWatchers.fs" />
     <Compile Include="Pipeline.fs" />
+    <Compile Include="MSBuildCrackerResolver.fs" />
     <Compile Include="BuildalyzerCrackerResolver.fs" />
```

Comment:
I just saw that there is a typo in the name of the File.

The file is named `BuildalyzerCrackerResolver` instead of `BuildanalyzerCrackerResolver`

Reply:
Yes, but that is because the nuget package has a typo: https://github.com/daveaglick/Buildalyzer

Reply:
Oh right.

My brains always corrected the typo in my mind.

Then no action required on that side